definitions/infra-container/definition.yml

```diff
@@ -50,6 +50,21 @@ synthesis:
         entityTagName: k8s.clusterName
       k8s.namespace.name:
         entityTagName: k8s.namespaceName
+  - identifier: container.id
+    name: container.name
+    encodeIdentifierInGUID: false
```

Comment:
wouldn't the name be too long for us to use as identifier?

Reply:
+1, also for consistency with the others it's probably better to encode them.

Reply:
done